lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -435,6 +435,8 @@ def enable_disable_extensions(
             for extension, enable in extensions.items():
                 ordered_extensions[extension] = enable
 
+            self._configure_pgaudit(False)
```

Comment:
Doesn't seem to be necessary for PG 14, but to maintain charm lib compatibility.